packages/components/atoms/f-filter-pill/src/components/FilterPill.vue

```diff
@@ -36,14 +36,22 @@
                 data-test-id="filter-pill-label"
                 :tabindex="-1">
                 <tick-icon :class="$style['c-filterPill-icon']" />
+                <p
+                    v-if="screenReaderMessage"
+                    data-test-id="filter-pill-sr-msg"
+                    class="is-visuallyHidden">
+                    {{ screenReaderMessage }}
+                </p>
                 <span
                     :class="$style['c-filterPill-text']"
-                    data-test-id="filter-pill-text">
+                    data-test-id="filter-pill-text"
+                    :aria-hidden="screenReaderMessage ? true : null">
```

Comment:
I wonder if we can simplify it to 
```suggestion
                    :aria-hidden="!!screenReaderMessage">
```